salt/metalk8s/salt/master/files/salt-master-manifest.yaml.j2

```diff
@@ -143,3 +145,7 @@ spec:
       hostPath:
         path: '/etc/kubernetes/pki'
         type: Directory
+    - name: salt-master-log
+      hostPath:
+        path: '/var/log/salt/master'
+        type: FileOrCreate
```

Comment:
To be consistent with other I would add the full directory as it's the directory for salt it does not hurt IMO

Also not sure it's really relevant but to be sure maybe do the same for salt api